src/Data/Array.purs

```diff
@@ -560,7 +566,12 @@ foreign import concat :: forall a. Array (Array a) -> Array a
 -- | into a single, new array.
 -- |
 -- | ```purescript
--- | concatMap (split $ Pattern " ") ["Hello World", "other thing"] = ["Hello", "World", "other", "thing"]
+-- | concatMap (split $ Pattern " ") ["Hello World", "other thing"]
+-- | ```
```

Comment:
Sorry to nitpick but I think I prefer the style you used in e.g. `alterAt`, where it's still all contained in a single code block.